tests/config/locales.tests.ts

```diff
@@ -1,65 +1,46 @@
-import {readFile as fsReadFile, readdir as fsReadDir} from 'fs';
+import {readFileSync as fsReadFile, readdirSync as fsReadDir} from 'fs';
 import {resolve as resolvePath} from 'path';
 
-function readDir(dir) {
-    return new Promise<string[]>((resolve, reject) => {
-        fsReadDir(resolvePath(__dirname, dir), (err, files) => {
-            if (err) {
-                reject(err);
-                return;
-            }
-            resolve(files);
-        });
-    });
+function readDir(dir: string) {
+    return fsReadDir(resolvePath(__dirname, dir));
 }
 
-function readLocale(name) {
-    return new Promise<string>((resolve, reject) => {
-        fsReadFile(resolvePath(__dirname, '../../src/_locales/', name), {encoding: 'utf-8'}, (err, data) => {
-            if (err) {
-                reject(err);
-                return;
-            }
-            resolve(data);
-        });
-    });
+function readLocale(name: string) {
+    return fsReadFile(resolvePath(__dirname, '../../src/_locales/', name), {encoding: 'utf-8'});
 }
 
-test('Locales', async () => {
-    const files = await readDir('../../src/_locales');
-    const enLocale = await readLocale('en.config');
+describe('Locales', () => {
+    const files = readDir('../../src/_locales');
+    const enLocale = readLocale('en.config');
     const enLines = enLocale.split('\n');
-    const locales: string[] = [];
-    for (const file of files) {
-        const locale = await readLocale(file);
-        locales.push(locale);
-    }
 
-    function compareLinesToEnLocale(predicate: (en: string, loc: string) => boolean) {
-        return locales.every((loc, j) => {
-            const lines = loc.split('\n');
-            for (let i = 0; i < Math.min(lines.length, enLines.length); i++) {
-                if (!predicate(enLines[i], lines[i])) {
-                    console.error(`${files[j]}, line ${i + 1}`);
-                    return false;
-                }
+    function compareLinesToEnLocale(loc: string, predicate: (en: string, loc: string) => boolean) {
+        const lines = loc.split('\n');
+        for (let i = 0; i < Math.min(lines.length, enLines.length); i++) {
```

Comment:
Let's change this to a cache length loop.
`for (let i = 0, len =Math.min(lines.length, enLines.length); i < len; i++) {